### PR TITLE
fix(minimap): disable minimap for ArcGIS base map

### PR DESCRIPTION
**Pull request by mspivak-actionengine**

### CODE REVIEW

**Comment by dsavinov-actionengine** [comment]
@belom88 please review

File: src/components/toogle-switch/toggle-switch.tsx [hunk 1]
```diff
@@ -1,11 +1,7 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import { color_canvas_primary_inverted } from "../../constants/colors";
 
-const Switch = styled.div`
-  position: relative;
-  width: 28px;
-  height: 18px;
-`;
+const color_control_disabled = "#808080";
```
Comment (belom88) [inline src/components/toogle-switch/toggle-switch.tsx:4]: Move to colors.ts. I would name it `dim_canvas_tertiary`. Make sure it looks good on the light theme

File: src/components/toogle-switch/toggle-switch.tsx [hunk 2]
```diff
@@ -71,26 +65,51 @@ const Slider = styled.span`
...
   onChange,
   name = "",
   id = "",
   title = "",
-}) => {
+  disabled = false,
+}: ToggleSwitchProps) => {
   return (
-    <Switch>
+    <Switch disabled={disabled}>
```
Comment (belom88) [inline src/components/toogle-switch/toggle-switch.tsx:105]: Disabled button should not react on hover

**Review by belom88 — CHANGES_REQUESTED** [review]

File: src/components/toogle-switch/toggle-switch.tsx (hunk 1 above)
Comment (mspivak-actionengine) [reply src/components/toogle-switch/toggle-switch.tsx]: Done.

File: src/components/toogle-switch/toggle-switch.tsx (hunk 2 above)
Comment (mspivak-actionengine) [reply src/components/toogle-switch/toggle-switch.tsx]: Done.

**Review by belom88 — APPROVED** [review]

**Pull request merged**